Python/Product/Debugger.Concord/Proxies/Structs/PyFrameObject.cs

```diff
@@ -68,29 +68,26 @@ public PyFrameObject(DkmProcess process, ulong address)
         private static bool IsInEvalFrame(DkmStackWalkFrame frame) {
             var process = frame.Process;
             var pythonInfo = process.GetPythonRuntimeInfo();
+            var name = "PyEval_EvalFrameEx";
             ulong addr = 0;
-            if (pythonInfo.LanguageVersion <= PythonLanguageVersion.V35) {
-                if (frame.ModuleInstance == pythonInfo.DLLs.Python) {
-                    addr = pythonInfo.DLLs.Python.GetFunctionAddress("PyEval_EvalFrameEx");
-                }
-            } else {
-                if (frame.ModuleInstance == pythonInfo.DLLs.Python) {
-                    addr = pythonInfo.DLLs.Python.GetFunctionAddress("_PyEval_EvalFrameDefault");
-                }
+            if (pythonInfo.LanguageVersion > PythonLanguageVersion.V35) {
+                name = "_PyEval_EvalFrameDefault";
             }
-
+            addr = pythonInfo.DLLs.Python.GetFunctionAddress(name);
             if (addr == 0) {
                 return false;
             }
 
-            return frame.InstructionAddress.IsInSameFunction(process.CreateNativeInstructionAddress(addr));
+            var addressMatch = frame.InstructionAddress.IsInSameFunction(process.CreateNativeInstructionAddress(addr));
```

Comment:
I wasn't getting address matching even when the native frame was the actual `_PyEval_EvalFrameDefault` frame. Maybe something is messed up with the pdb because the RVAs were off.